lisa/features/hibernation.py

```diff
@@ -1,12 +1,17 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
+from typing import Any
 
 from lisa.feature import Feature
 
 FEATURE_NAME_HIBERNATION = "Hibernation"
 
 
 class Hibernation(Feature):
+    @classmethod
+    def on_before_deployment(cls, *args: Any, **kwargs: Any) -> None:
+        cls._install_by_platform(*args, **kwargs)
```

Comment:
It doesn't need to follow GPU feature, the GPU feature is a reference.

This method can do some basic check, for example, only the feature setting is enabled, then call the `_enable_hibernation` (the `_install` is for GPU driver, hibernation doesn't need to `install`).